action.yml

```diff
@@ -62,6 +62,10 @@ runs:
           VERSION: ${{ inputs.version }}
           SHA256SUM: ${{ inputs.sha256sum }}
         run: |
+          if [ "$VERSION" = "latest" ]; then
+            VERSION=$(curl -s https://pkgs.tailscale.com/stable/ | grep -o 'value="[0-9.]*"' | sed 's/value="//;s/"//' | sort -V | tail -n 1)
```

Comment:
```
VERSION=$(curl -s "https://pkgs.tailscale.com/stable/?mode=json" | jq -r .Version)
```

I'm fairly certain the GitHub Actions image has jq installed.

Reply:
Because it returns the correct canonical version rather than scrapping the webpage for potentially a newer patch that only exists for a specific platform.

Regardless it's been stated here https://github.com/tailscale/github-action/issues/28#issuecomment-1838519627 and is now officially published in the documentation https://tailscale.com/kb/1276/tailscale-github-action?q=action#tailscale-github-action-version that the intent is specifically not to pull the latest version of Tailscale with the action.

In which case this PR should be closed

Reply:
Thank you, where can I find the documentation on the `https://pkgs.tailscale.com/stable/?mode=json` API?  I don't see where it's documented that it returns the canonical version.

Reply:
It's not publicly documented, it's just an internal API of the pkg server.